pkg/controller/cache/claim.go

```diff
@@ -35,18 +35,11 @@ import (
 	"github.com/crossplaneio/stack-azure/apis/cache/v1beta1"
 )
 
-// A RedisClaimSchedulingController reconciles RedisCluster claims that include
-// a class selector but omit their class and resource references by picking a
-// random matching Azure Redis class, if any.
-type RedisClaimSchedulingController struct{}
-
-// SetupWithManager sets up the RedisClaimSchedulingController using the
-// supplied manager.
-func (c *RedisClaimSchedulingController) SetupWithManager(mgr ctrl.Manager) error {
-	name := strings.ToLower(fmt.Sprintf("scheduler.%s.%s.%s",
-		cachev1alpha1.RedisClusterKind,
-		v1beta1.RedisKind,
-		v1beta1.Group))
+// SetupRedisClaimScheduling adds a controller that reconciles RedisCluster
+// claims that include a class selector but omit their class and resource
+// references by picking a random matching Azure Redis class, if any.
+func SetupRedisClaimScheduling(mgr ctrl.Manager, l logging.Logger) error {
+	name := claimscheduling.ControllerName(v1beta1.RedisKind)
```

Comment:
I think we made the decision [here](https://github.com/crossplaneio/stack-gcp/pull/152/files) to use the name of the claim type for the controller name.

Reply:
There are some other instances of this that I did not add a comment to in order to avoid comment overload :)